7.2.Цикл_for:_функция_range/7.2.7.py

```diff
@@ -1,6 +1,5 @@
-# Последовательность чисел 1
-
+# put your python code here
 m = int(input())
 n = int(input())
 for i in range(m, n+1):
```

Comment:
не забываем про PEP8:
```suggestion
for i in range(m, n + 1):
```